src/popup/components/InputField.vue

```diff
@@ -59,7 +59,6 @@
             autocomplete="off"
             step="any"
             data-cy="input"
-            :type="type"
```

Comment:
Are you sure? The input should have the type property. Here we are using the component's property `type`.